internal/cmd/util/factory.go

```diff
@@ -45,18 +45,18 @@ func (f *FactoryImpl) Clock() Clock {
 
 // RESTClient returns the rest client for the garden cluster, identified by the garden cluster identity
 func (f *FactoryImpl) RESTClient(gardenClusterIdentity string) (rest.Interface, error) {
-	config := &GardenloginConfig{}
+	config := &Config{}
```

Comment:
do we really need to unmarshal the config for each new client instance. Couldn't this be done in an init func?

Reply:
I moved it to to a separate function to load it lazily, once

Reply:
https://github.com/gardener/gardenlogin/issues/22